Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM node:14-alpine3.10
+
+ENV NODE_VERSION 14.9.0
+
+# Add package.json before rest of repo for caching
+ADD package.json /app/
+
+WORKDIR /app
+
+RUN npm install --silent
+
+ADD . /app
+
+# start app
+CMD ["npm", "start"]
```

Comment:
this start is development, you need to pass arg NODE_ENV = production

Reply:
we can handle env by ci tool like `travis`

Reply:
@vutoan266 can we switch to use `github action`, so that this lib will be depended on 1 single source (github)?